peep.c

```diff
@@ -2690,7 +2690,7 @@ Perl_rpeep(pTHX_ OP *o)
 {
     OP* oldop = NULL;
     OP* oldoldop = NULL;
-    OP** defer_queue[MAX_DEFERRED]; /* small queue of deferred branches */
+    OP** defer_queue[MAX_DEFERRED] = { NULL, NULL, NULL, NULL }; /* small queue of deferred branches */
```

Comment:
This could be shortened to
```c
OP** defer_queue[MAX_DEFERRED] = { NULL };
```
This decouples the initializer from the value of `MAX_DEFERRED`.

Reply:
Ack. Thanks.

Reply:
Fixed with the most recent push.